.env.local

```
@@ -0,0 +1,3 @@
+DB_URL=jdbc:postgresql://localhost:5433/deokhugam
+DB_USERNAME=deokhugam
+DB_PASSWORD=deokhugam
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_URL key

(UnorderedKey)